flutter/lib/src/web/web_sentry_js_binding.dart

```diff
@@ -22,6 +22,22 @@ class WebSentryJsBinding implements SentryJsBinding {
     _client = SentryJsClient();
   }
 
+  @override
+  void updateSession({int? errors, String? status}) {
+    final isolationScope = SentryJsIsolationScope();
+    JSObject? currentSession = isolationScope.getSession();
+
+    if (status != null) {
```

Comment:
If we never set null for errors/status, could we make them non-optional?

Reply:
you could theoreticaly also set only status for example, also keeping them null makes it a bit easier to work with mockito